docs/components/DocumentationHeader.tsx

```diff
@@ -318,14 +325,9 @@ export default class DocumentationHeader extends React.PureComponent<Props> {
             <div css={STYLES_LOGO_CONTAINER}>
               <Link href="/" passHref>
                 <a css={STYLES_UNSTYLED_ANCHOR}>
-                  <span css={STYLES_LOGO}>
+                  <span css={STYLES_LOGO} aria-hidden>
```

Comment:
ℹ️ The modification gather the Expo Icon and the "Expo" word to be under the same link, reducing the tab sequences by one. Instead of having to tab from the logo to the icon, we can directly tab to both of them at the same time.

Before:

![Navigating the expo navbar shows the icon is focusable](https://user-images.githubusercontent.com/3874873/119312175-4ed7ef00-bc72-11eb-9d91-2ba5a32c7513.gif)

After:

---

Also, it prevents the expo icon from being announced by assistive technology.

![Navigating expo navbar shows that the icon is not focusable anymore](https://user-images.githubusercontent.com/3874873/119312233-61522880-bc72-11eb-83a3-3b7dba310761.gif)